src/firefly/js/ui/SearchPanel.jsx

```diff
@@ -114,7 +115,7 @@ function SearchForm({searchItem, style, initArgs}) {
     );
 }
 
-function SideBar({activeSearch, groups=[]}) {
+export function SideBar({activeSearch, groups=[]}) {
```

Comment:
Who is using these exports?

Reply:
The `SearchForm` and `SideBar` exports were unused, removed them (I think those were for the `SearchPanelRouter` component I had previously created, but forgot to remove the exports while cleaning that up)

Reply:
I see you also remove Search.jsx.  But, you forgot to remove the classNames that was added to euclid.css.